modules/ref_hardware-compatibility-with-irq-affinity.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: REFERENCE
+[id="ref_hardware-compatibility-with-irq-affinity_{context}"]
+= Hardware compatibility with IRQ affinity
+
+For core isolation, all the hardware components of your server must support IRQ affinity. To check if the hardware components of your server support IRQ affinity, view the server's hardware specifications or contact your hardware provider. 
```

Comment:
For minimalism: s/all the hardware components of your server/all server hardware components